packages/loader/container-loader/src/containerContext.ts

```diff
@@ -97,9 +96,7 @@ export class ContainerContext implements IContainerContext {
 
     /** @deprecated Added back to unblock 0.56 integration */
     public get id(): string {
-        const resolvedUrl = this.container.resolvedUrl;
-        ensureFluidResolvedUrl(resolvedUrl);
-        return resolvedUrl.id;
+        return this.container.resolvedUrl?.id;
```

Comment:
this didn't work for me because resolvedUrl could be IWebResolvedUrl which does not have id property